src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
![image](https://github.com/mate-academy/jv-salary-info/assets/20326858/0a111390-cd69-4b1c-b60a-7c5da052ec77)

Reply:
Im read before checklist. I didnt see this line

Reply:
The reason is to make students pay attention to the task details.
Is it painful to have a one-line rejection? Yes. And I am sorry. But there is no other way. It will protect you from spending tens of hours writing wasteful code in the future.

Reply:
Okay